web/src/AnalyticsNudge.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react"
+import "./AnalyticsNudge.scss"
+
+class AnalyticsNudge extends Component {
+  render() {
+    return (
+      <div className="AnalyticsNudge" key="AnalyticsNudge">
+        <div>
+          Congrats on your first Tilt resource 🎉 Help us help you: may we
+          collect anonymized data on your usage. (Read more{" "}
```

Comment:
Should this end in a question mark?

> may we collect anonymized data on your usage?

Reply:
...GEEZ DAN THAT'S WHAT I SAID THE FIRST TIME WEREN'T YOU LISTENING 👀